doc/source/ray-core/actors/named-actors.rst

```diff
@@ -0,0 +1,157 @@
+Named Actors
+============
+
+An actor can be given a unique name within their :ref:`namespace <namespaces-guide>`.
+This allows you to retrieve the actor from any job in the Ray cluster.
+This can be useful if you cannot directly
+pass the actor handle to the task that needs it, or if you are trying to
+access an actor launched by another driver.
+Note that the actor will still be garbage-collected if no handles to it
+exist. See :ref:`actor-lifetimes` for more details.
+
+.. tabbed:: Python
+
+    .. code-block:: python
+
+        # Create an actor with a name
+        counter = Counter.options(name="some_name").remote()
```

Comment:
can we make this executable? At least it's missing the import. If we want "obvious" pseudo-code, that's ok sometimes, but it's better to aim for copypasta.